charts/qdrant/templates/configmap.yaml

```
@@ -7,6 +7,7 @@ metadata:
 data:
   initialize.sh: |
     #!/bin/sh
+    ulimit -n 65536
```

I think we should pick a bigger default number, just in case. It's definitely possible for some big deployments to reach the 65k limit.

```suggestion
    ulimit -n 1048576
```